src/ontology/mondo.Makefile

```diff
@@ -409,6 +409,22 @@ update-rare-subset:
 ##### Mondo External Content Pipeline ################
 ######################################################
 
+####################################
+##### OMIM #####################
+####################################
+
+.PHONY: update-omim-genes
+update-omim-genes:
+	$(MAKE) $(TMPDIR)/external/processed-mondo-omim-genes.robot.owl -B
+	# We need to be less aggressive here, as some gene relations were not originally sourced
+	# from OMIM, and were added, for example, for ClinGen.
```

Comment:
From @sabrinatoro definitely keep clinicalgenome, PMID and orcid annotated genes.